docs/administering_lagoon/rbac.md

```diff
@@ -0,0 +1,128 @@
+# Role Based Access Control
+
+Version 1.0 of Lagoon changed how you access your projects!  Access to your project is handled via groups, with projects assigned to one or multiple groups. Users are added to groups with a role. Groups can also be nested within subgroups. This change provides a lot more flexibility and the possibility to recreate real world teams within Lagoon.
+
+## Roles
+
+When assigning a user to a group, you need to provide a role for that user inside this group. Each one of the 5 current existing roles gives the user different permissions to the group and projects assigned to the group.
+
```

Comment:
Let's add a short sentence explanation of all the roles in here, so it's a bit easier to understand the different roles:

**Platform Wide Roles**
- `Platform Wide Admin` - has access to everything across all of Lagoon, also to dangerous mutations like deleting all projects. Use very very very carefully
- `Platform Wide Owner` - has access to every Lagoon Group like the Role `Owner`, can be used if you need a user that needs access to everything but you don't want to assign the user to every Group.

**Group Roles**
- `Owner` - Can do everything within a Group and it's associated Projects. Can add manage users of group. Be careful with this role, it can delete projects and production environments! 
- `Maintainer` - Can do everything within a Group and it's associated Projects **except** deleting the project itself or the production environment. Can add manage users of group.
- `Developer` - Role with ssh access only to development environments. Cannot access, update or delete the production environment. Can run a sync task with the production environment as a source, but not as the destination. Cannot manage users of Group. - **IMPORTANT:** this Role does not prevent the deployment of the production environment as a deployment is triggered via a Git Push you need to make sure that your Git server prevents these users to push into the branch defined as production environment.
- `Reporter` - View access only, cannot ssh access any environments or make modifications on them. Can run Cache Clear Task. Mostly used for Stakeholders to have access to Lagoon UI and Logging.
- `Guest` - Same role as `Reporter`

Reply:
They have been added!